app/_landing_pages/gateway/secrets-management.yaml

```
@@ -138,11 +138,12 @@ rows:
             blocks:
               - type: text
                 text: |
-                  There are two main ways to rotate secrets:
+                  {{site.base_gateway}} can't rotate secrets, it can only retrieve rotated secrets. You must configure secret rotation in your third-party vault settings.
```

```suggestion
                  If you have secrets rotation configured in your third-party vault settings, {{site.base_gateway}} can retrieve rotated secrets on a schedule.
```
Trying to frame it in a less negative sounding way. Make sure to check my indentation here, I can't tell if this is right.